programs/keeper-client/Commands.cpp

```diff
@@ -113,13 +113,21 @@ bool CreateCommand::parse(IParser::Pos & pos, std::shared_ptr<ASTKeeperQuery> &
     int mode = zkutil::CreateMode::Persistent;
 
     if (ParserKeyword(Keyword::PERSISTENT).ignore(pos, expected))
-        mode = zkutil::CreateMode::Persistent;
+    {
+        ParserToken{TokenType::Whitespace}.ignore(pos);
```

Comment:
TokenIterator should be capable of ignoring whitespaces and comments.

Reply:
I will check again, but I remember that if I removed this line then parsing doesn't work as expected.

Reply:
Yes, removing this line will lead to error, I don't understand yet.

```
03161_clickhouse_keeper_client_create_sequential:                       [ FAIL ] - having stderror:
Syntax error: failed at position 67 ('SEQUENTIAL'):

create /test-keeper-client-test_y7xdrqax/tmp- 'foobar0' EPHEMERAL SEQUENTIAL; get /test-keeper-client-test_y7xdrqax/tmp-0000000000

Expected end of query
```

Reply:
> But since then, the parsing method has changed, and now I actually don't see any reason why it's still turned off

OK, disregard that. The problem with the whitespace is when we are trying to parse a query like
`create /foo/bar/ value`, it will contain seven tokens without whitespaces:
`create`
`/`
`foo`
`/`
`bar`
`/`
`value`

So, we are unable to understand where the path ends unless we force a user to put paths into semicolons. 
That's why we shouldn't ignore whitespaces automatically and just remove them manually.